source/appModules/shellexperiencehost.py

```diff
@@ -16,3 +31,14 @@ def chooseNVDAObjectOverlayClasses(self,obj,clsList):
 				clsList.remove(ContentGenericClient)
 			except ValueError:
 				pass
+
+	# Argh, somehow, item status property repeats when Action Center is opened more than once.
+	_itemStatusMessage = None

```

Comment:
May be add the word cached to this variable to make it clear that this is used for comparison purposes.